src/mavis/summary/summary.py

```diff
@@ -190,45 +192,120 @@ def group_by_distance(calls, distances):
     """
     for chrom in dgv_regions_by_reference_name:
         dgv_regions_by_reference_name[chrom] = sorted(
             dgv_regions_by_reference_name[chrom], key=lambda x: x.start
         )
-
     lowest_resolution = max([len(b.break1) for b in bpps])  # only need start res
-
+    for bpp in bpps:
+        bpp.data[COLUMNS.dgv] = []
+        bpp.data[COLUMNS.known_sv_count] = 0
     # only look at the bpps that dgv events could pair to, Intrachromosomal
     for bpp in [
         b for b in bpps if not b.interchromosomal and b.break1.chr in dgv_regions_by_reference_name
     ]:
-        bpp.data['dgv'] = []
         for dgv_region in dgv_regions_by_reference_name[bpp.break1.chr]:
             dist = abs(Interval.dist(Interval(dgv_region.start), bpp.break1))
-            if dist > lowest_resolution + distance:
+            if dist > lowest_resolution + input_cluster_radius:
                 continue
             elif (
-                dist > distance
-                or abs(Interval.dist(Interval(dgv_region.end), bpp.break2)) > distance
+                dist > input_cluster_radius
+                or abs(Interval.dist(Interval(dgv_region.end), bpp.break2)) > input_cluster_radius
             ):
                 continue
             refname = dgv_region.reference_object
             try:
                 refname = dgv_region.reference_object.name
             except AttributeError:
                 pass
-            bpp.data['dgv'].append(
+            bpp.data[COLUMNS.dgv].append(
                 '{}({}:{}-{})'.format(dgv_region.name, refname, dgv_region.start, dgv_region.end)
             )
-        bpp.data['dgv'] = ';'.join(bpp.data['dgv'])
+            bpp.data[COLUMNS.known_sv_count] = len(bpp.data[COLUMNS.dgv])
+        bpp.data[COLUMNS.dgv] = ';'.join(bpp.data[COLUMNS.dgv])
+
+
+def annotate_dgv(
+    bpps: List[BreakpointPair],
+    dgv_regions_by_reference_name: Dict[Tuple[str], List[BreakpointPair]],
+    input_cluster_radius,
+):
+    """
+    Given a list of breakpoint pairs (bpps) and bpps from another reference set), annotate the events that are within the set distance of both breakpoints
+
+    Args:
+        bpps (list) : the list of BreakpointPair objects
+        dgv_regions_by_reference_name (dict) : tuple of (break1_chr,break2_chr) and its associated list of BreakpointPair/BioInterval objects specified by the MAVIS input file
+        cluster_radius (int) : Distance used in matching input SVs to reference SVs through clusterind, defined by summary.cluster_radius in the configuration file
+    """
+
+    if isinstance(list(dgv_regions_by_reference_name.values())[0][0], BreakpointPair):
+        annotate_with_dgv_bpp(
+            bpps, list(chain(*dgv_regions_by_reference_name.values())), input_cluster_radius
+        )
+
+    elif isinstance(list(dgv_regions_by_reference_name.values())[0][0], BioInterval):
```

Comment:
just use else here, otherwise if they enter something we don't expect the function does nothing, we would rather it tries and fails

Reply:
Addressed in [0e7aad3](https://github.com/bcgsc/mavis/pull/336/commits/0e7aad362a9cbf50adc2651df62a218dd698c7ff).